bot_reddit.py

```diff
@@ -0,0 +1,21 @@
+import praw
+import creds
+
+r = praw.Reddit(
```

Comment:
Careful about instantiating this globally. Can the session expire? Will you ever want to fetch a second subreddit, or sign in with different username (to access private subreddits)? Probably will work here, but concerns to be mindful of.